home/src/main/resources/rdf/dynapiAbox/everytime/serialization_types.n3

```diff
@@ -0,0 +1,298 @@
+        dynapi:methodName 		"serialize";
+        dynapi:methodArguments	"input";
+        dynapi:isStaticMethod	"true"^^xsd:boolean ;
+        vitro:mostSpecificType  dynapi:ImplementationConfig .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/deserialization-config/json-node> 
+        a                       dynapi:ImplementationConfig ;
+        rdfs:label              "Fasterxml Json node deserialization config"@en-US ;
+        dynapi:className 	"edu.cornell.mannlib.vitro.webapp.dynapi.data.implementation.JsonFasterxmlNode" ;
+        dynapi:methodName 	"deserialize";
+        dynapi:methodArguments	"input";
+        dynapi:isStaticMethod	"true"^^xsd:boolean ;
+        vitro:mostSpecificType  dynapi:ImplementationConfig .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/serialization-config/uri-resource>
+        a                       dynapi:ImplementationConfig ;
+        rdfs:label              "Uri resource serialization config"@en-US ;
+        dynapi:className 		"org.apache.jena.rdf.model.Resource" ;
+        dynapi:methodName 		"toString";
+        dynapi:methodArguments	"";
+        dynapi:isStaticMethod	"false"^^xsd:boolean ;
+        vitro:mostSpecificType  dynapi:ImplementationConfig .
+        
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/deserialization-config/uri-resource>
+        a                       dynapi:ImplementationConfig ;
+        rdfs:label              "Uri resource deserialization config"@en-US ;
+        dynapi:className        "org.apache.jena.rdf.model.impl.ResourceImpl" ;
+        dynapi:methodName       "";
+        dynapi:methodArguments  "input";
+        dynapi:isStaticMethod   "true"^^xsd:boolean ;
+        vitro:mostSpecificType  dynapi:ImplementationConfig .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/serialization-config/string-literal>
+        a                       dynapi:ImplementationConfig ;
+        rdfs:label              "String literal serialization config"@en-US ;
+        dynapi:className 	"org.apache.jena.rdf.model.impl.LiteralImpl" ;
+        dynapi:methodName 	"toString";
+        dynapi:methodArguments	"";
+        dynapi:isStaticMethod	"false"^^xsd:boolean ;
+        vitro:mostSpecificType  dynapi:ImplementationConfig .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/deserialization-config/string-literal>
+        a                       dynapi:ImplementationConfig ;
+        rdfs:label              "String literal deserialization config"@en-US ;
+        dynapi:className 	"org.apache.jena.rdf.model.ResourceFactory" ;
+        dynapi:methodName 	"createStringLiteral";
+        dynapi:methodArguments	"input";
+        dynapi:isStaticMethod	"true"^^xsd:boolean ;
+        vitro:mostSpecificType  dynapi:ImplementationConfig .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/deserialization-config/generated-uri-resource> 
```

Comment:
Here we need only deserialization-config, right? No scenario for need for serialization.